contrib/message-capture/message-capture-docs.md

```diff
@@ -0,0 +1,25 @@
+# Per-Peer Message Capture
+
+## Purpose
+
+This feature allows for message capture on a per-peer basis.  It answers the simple question: "Can I see what messages my node is sending and receiving?"
+
+## Usage and Functionality
+
+* Run `bitcoind` with the `-capturemessage` option.
+* Look in the `message_capture` folder in your datadir.
+  * Typically this will be `~/.bitcoin/message_capture`.
+  * See that there are many folders inside, one for each peer names with its IP address and port.
+  * Inside each peer's folder there are two `.dat` files: one is for received messages (`msgs_recv.dat`) and the other is for sent messages (`msgs_sent.dat`).
+* Run `contrib/message-capture/message-capture-parser.py` with the proper arguments.
+  * See the `-h` option for help.
+  * To see all messages, both sent and received, for all peers use:
+    ```
+    python contrib/message-capture/message-capture-parser.py -o out.json \
```

Comment:
nit: I'd just drop the `python` command part and call the script directly (also because on some systems, `python` symlinks to `python2`, not `python3`! See also https://www.python.org/dev/peps/pep-0394/), thanks to the she-bang on top of the file the right python interpreter will be called.

Reply:
Thank you again for the link.  I'm so used to having python environment problems that I've always tried to be as explicit as possible when running python scripts.  Since switching to conda I haven't had any problems, and I'm sure other people have things figured out as well.  (Either they only have one python install or they use an environment manager too)  I'll make this change.